lib/entangler/executor/validation/master.rb

```diff
@@ -10,6 +10,7 @@ def validate_opts
           super
           if @opts[:remote_mode]
             @opts[:remote_port] ||= '22'
+            @opts[:config] ||= '$HOME/.ssh/config'
```

Comment:
Feels like it would be better to default to nil and then only pass the option to the ssh command if this is provided? Otherwise there is no way to opt out of this - should there be?

Reply:
Ya I was rushing this, will action.

Reply:
https://github.com/daveallie/entangler/compare/ccf8080ad139bf5dc6a3531cad86d0b2412e6fd0..c76f7d060f848466fdada6e392ac3e815157db92

Reply:
QA'ed with the latest changes ✅